charts/tekton-pipelines/values.yaml

```diff
@@ -190,12 +190,21 @@ buildpacks:
           workingDir: $(resources.inputs.app.path)
           script: |
             #!/usr/bin/env bash
-            set -e
-            [ -f manage.py ] && (cp requirements/$(params.environment)*.txt requirements.txt \
-              || echo "Unable to find requirements for $(params.environment)")
-            [ -f requirements.txt ] && echo "Found requirements.txt" \
-              || (echo "requirements.txt is not found! Halting execution."; exit 1;)
-
+            set -eo pipefail
+            if [ -f poetry.lock ]; then
+              echo "poetry.lock is found, skipping pip section."
+              exit
```

Comment:
let's come up with less nested ifs, use `exit 0` to prevent getting in the code below

Reply:
Done